### DISCUSSION TITLE
Working with external scripts to process files

### DISCUSSION THREAD

**Discussion by Murmele** [typst/typst#1292] [Ideas]
Hello,

currently I am using latex to write my documents. A lot of images are drawn with draw.io and therefore not directly available to insert into the document. Instead of calling directly pdflatex I am using a makefile to automatically converting the draw.io files to pdf files and then calling pdflatex.
This works really good, because due to the makefile, the images are only converted once they changed and I never have a outdated duplicate of the image.

Is there a plan for something similar to this workflow?
Of course currently this is also possible with a makefile and typst, but in this case live update would not work anymore.

**Comment by laurmaedje** [comment]
Perhaps this is of interest: #1056

**Reply by Murmele** [reply]
Thank you for the response. So I will close this one
